### PR TITLE
UI neutrality

### PR DESCRIPTION
**Pull request by OscarSpruit**
## Description
- Replace flags with ISO codes
- Change related strings

## Checklist 
- [x] Changes are tested manually
- [x] Add relevant labels to PR  

COAND-849

### CODE REVIEW

File: ui-core/src/main/res/values-hr-rHR/strings.xml [hunk 1]
```diff
@@ -51,8 +51,8 @@
     <string name="checkout_address_lookup_initial">Ne možeš potražiti svoju adresu?</string>
     <string name="checkout_address_lookup_initial_description">Uvijek možete #ručno unijeti svoju adresu#</string>
     <string name="checkout_address_lookup_empty">Nisu pronađeni rezultati</string>
-    <string name="checkout_address_lookup_empty_description">\'%s\' nije se podudarao s ničime, pokušajte ponovno ili upotrijebite #ručni unos adrese#</string>
+    <string name="checkout_address_lookup_empty_description">&#39;%s&#39; nije se podudarao s ničime, pokušajte ponovno ili upotrijebite #ručni unos adrese#</string>
```
Comment (jreij) [inline ui-core/src/main/res/values-hr-rHR/strings.xml:54]: Is this change intentional?

File: ui-core/src/main/res/values-de-rDE/strings.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
   -->
 
 <resources>
-    <string name="pay_button">Zahlen</string>
+    <string name="pay_button">Zahle</string>
```
Comment (jreij) [inline ui-core/src/main/res/values-de-rDE/strings.xml:10]: Is this change intentional?

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -10,3 +10,7 @@
 
 ## Fixed
 - Fixed various memory leaks.
+
+## Changed
+- Flags are replaced by ISO codes in the phone number inputs.
```
Comment (jreij) [inline RELEASE_NOTES.md:15]: Can we mention which payment methods are affected by this?

**Review by ozgur00 — APPROVED** [review]
